modules/core/index.js

```diff
@@ -59,3 +61,4 @@ systems.available.set('ui', UiSystem);
 systems.available.set('uploader', UploaderSystem);
 systems.available.set('urlhash', UrlHashSystem);
 systems.available.set('validator', ValidationSystem);
+systems.available.set('colors', ColorSystem);
```

Comment:
Ah yes! Since this export is adding the color system to 'Systems.available', that means it will be auto-started by the Context init code (see my earlier note).

Reply:
Oh I think this answers my above question